### DISCUSSION TITLE
msnoise plot dvv +msnoise 1.6 + How to generate daily mean and median results?

### DISCUSSION THREAD

**Discussion by Amethyst4** [ROBelgium/MSNoise#391] [General]
I generated the image using the command msnoise plot dvv. I would like to understand how the mean and median values in the plot are calculated, especially considering that there is a result for every single day.

I know that -M represents dvv, and I can extract the M values for each station pair from the DTT folder to plot the time evolution of dvv for individual station pairs. However, these data are not continuous and do not provide daily results.

How does the msnoise plot dvv command handle this to generate daily data for the mean and median? Could you please clarify the process or provide further details about the interpolation or resampling method used?
![Figure_1](https://github.com/user-attachments/assets/36de79e2-9eaf-4c9c-86cf-211fea3b83c0)

**Comment by ThomasLecocq** [comment]
In the msnoise 1.6.x series, the data for different mov-stacks "extended" the data over data gaps if there were enough days available before, which is not always desired. The dvv results are not "interpolated" from the DTT results, just plotted. The DTT come from the MWCS that have been calculated from the "extensions" at the stack -m step.

This is not ideal, in the new msnoise 2 version will most probably change that default behaviour by masking "no-data days".

**Reply by Amethyst4** [reply]
Thank you for your response. I have another question regarding cross-correlation computation. If I want to compute the cross-correlation for a specific frequency range, how should I configure it? I only found high-pass and low-pass filtering options related to data preprocessing (preprocess_lowpass and preprocess_highpass) in the cross-correlation computation configuration parameters. Additionally, is it possible to perform cross-correlation calculations for multiple different frequency bands simultaneously?

**Reply by ThomasLecocq** [reply]
Yes, as explained in the documentation and the original SRL article, many filters can be defined in the filters table.
